lms/envs/common.py

```diff
@@ -1654,6 +1654,14 @@
     'ROOT_PATH': '/tmp/edx-s3/ora2-responses',
 }
 
+##################### ORA2 Responses Download #######################
```

Comment:
So really all reports go into the settings specified in GRADES_DOWNLOAD right now. Not sure if it's really that big a deal.